src/variables.jl

```diff
@@ -2816,6 +2832,15 @@ con : 5 x ≤ 2
 
 julia> normalized_coefficient(con, x)
 5.0
+
+julia> @constraint(model, con_vec, [x, 2x + 1, 3] >= 0)
+con_vec : [x, 2 x + 1, 3] ∈ MathOptInterface.Nonnegatives(3)
+
+julia> normalized_coefficient(con_vec, x)
+3-element Vector{Float64}:
+ 1.0
+ 2.0
+ 0.0
```

Comment:
Another option is whether the return for this should be `[(1, 1.0), (2, 2.0)]` to match the expectation of `set_normalized_coefficient`.

Reply:
I would rather have a `SparseArrays.SparseVector`

Reply:
I don't really want a `SparseVector`. This would be perhaps the only place in all of JuMP that returns such an object. We should either return `Vector` or return the input needed by `set_normalized_coefficient`.

Reply:
The same input as `set_normalized_coefficient` then I'd say

Reply:
Okay, how's this?

Reply:
It makes sense that `set_normalized_coefficient(c, x, normalized_coefficient(c, x))` preserves the current model.